libs/windy-sounding/types/client/idbInstances.d.ts

```
@@ -0,0 +1,71 @@
+import { IDB } from '@windy/IDB';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)